AwesomeCalculator/app/src/main/java/com/example/drcreeper/awesomecalculator/activities/MainActivity.java

```diff
@@ -0,0 +1,42 @@
+package com.example.drcreeper.awesomecalculator.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+
+import com.example.drcreeper.awesomecalculator.R;
+import com.example.drcreeper.awesomecalculator.asynktasks.DeleteHistoryListAsyncTask;
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+    }
+/*
```

Comment:
Do not keep commented code in repository. It will stale when code around commented fragment will be refactored and updated. Also we have history in git if we want to access it.